components/mpas-ocean/src/shared/mpas_ocn_time_average_coupled.F

```diff
@@ -311,6 +343,48 @@ subroutine ocn_time_average_coupled_accumulate(statePool, forcingPool, timeLevel
            !$omp end parallel
         end if
 
+        ! Accumulate polar fields if necessary
+        if(trim(config_land_ice_flux_mode) == 'standalone' .or. 'data') then
```

Comment:
```suggestion
        if(trim(config_land_ice_flux_mode) == 'standalone' .or. trim(config_land_ice_flux_mode) == 'data') then
```
I think this actually explains the zeros below.

Reply:
thanks for pointing that out @xylar . I've made those changes, and also tried just reading in `landIceFreshwaterFlux` rather than `dataLandIceFreshwaterFlux`, but still getting 0s in `CRYO1850-DISM` cases. 
An unelegant solution would be to go back to the previous commit's way for data, since those won't change over a coupling interval, but that's not ideal. I'll try to sort this out here.

Reply:
Hmm, that's too bad. Keep us posted and I can dig into it if need be.

Reply:
> An unelegant solution would be to go back to the previous commit's way for data, since those won't change over a coupling interval, but that's not ideal.

I'm not really okay with that solution.  For one, it isn't unlikely that we might switch to a time-varying version of the data forcing at some point.